gitstats.py

```diff
@@ -809,6 +809,7 @@ def create(self, data, path):
 """)
         ### Authors
         data_file.write('authors_cols = ['
+                '{formatter: "rownum", width: 40 },'
```

Comment:
This doesn't appear to work properly when sorting; the number doesn't match the initial sort order, but always has `1` at the top

Reply:
![image](https://user-images.githubusercontent.com/425260/181846917-790b7d76-749e-4ebf-b399-930e8670b26c.png)

Reply:
@eric-wieser That's exactly the point. This way we can see who has the 78th most lines added by sorting by that column and scrolling down.